core/server/master/src/main/java/alluxio/master/meta/DefaultMetaMaster.java

```diff
@@ -352,6 +352,18 @@ public ConfigCheckReport getConfigCheckReport() {
     return pathConfig;
   }
 
+  @Override
+  public String getConfigurationVersion() {
+    // TODO(cc)
```

Comment:
Will this be implemented in another PR, is there significant logic here?

Reply:
yes, it's in another PR, core logic is in https://github.com/Alluxio/alluxio/pull/8896.

Reply:
Got it thanks